### ISSUE TITLE
404 page does not support i18n

### ISSUE THREAD

**Issue by ludanxer** [vuejs/vuepress#1987]
- [x] I confirm that this is an issue rather than a question.

## Bug report

I just find this on [stackoverflow](https://stackoverflow.com/questions/58579031/is-translation-supposed-to-be-working-for-notfound-vue), and test it on the official website

#### Steps to reproduce

Try to visit [vuepress.vuejs.org/zh/test](https://vuepress.vuejs.org/zh/test)

#### What is expected?

A 404 page with Chinese and

```html
<html lang="zh-CN" >
 ```

#### What is actually happening?

```html
<html lang="en-US" >
 ```

#### Other relevant information

- Output of `npx vuepress info` in my VuePress project:

**Comment by dzetah** [comment]
Hello @ludanxer, this seems like a good fist issue for me on this project !
I will submit a PR for it later this week ;)

**Comment by ThaddeusJiang** [comment]
Hi @dzetah  
Did you submit a PR?

**Comment by ludanxer** [comment]
Yes, he kindly did #2004 , I believe it's not ready because

> Currently, it will show the default messages first, and change to the locale messages.
> 
> It's not so perfect 🤔
> 
> https://deploy-preview-2004--vuepress.netlify.com/zh/guide/aaa
> 
> ![demo-locale](https://user-images.githubusercontent.com/18205362/70528999-7c1e0380-1b8a-11ea-841a-5e919de03fed.gif)

**Comment by dzetah** [comment]
> Hi @dzetah  
> 
> Did you submit a PR?

Yes I did but I have to fix it because there’s a still a small bug. I had no time at all to do that recently, I’ll try to do it this week.

**Comment by faraz16iqbal** [comment]
Hey, is this issue still open?
